src/CrashlogFile.spec.ts

```diff
@@ -137,6 +140,255 @@ describe('CrashlogFile', () => {
         });
     });
 
+    describe('parseCrashes', () => {
+        describe('parses the hardware platform section', () => {
+            it('empty section', () => {
+                expect(file.parseHardwarePlatformSection([])).to.be.empty;
+                expect(file.parseHardwarePlatformSection(['', ''])).to.be.empty;
+                expect(file.parseHardwarePlatformSection(['', '', ''])).to.be.empty;
+            });
+
+            it('contains underscores', () => {
+                expectContainSubset(
+                    // eslint-disable-next-line no-tabs
```

Comment:
You can use `\t` instead a literal tap to acomplish this (and then remove the eslint disable comment)